bootstrap/bootstrap.md

```diff
@@ -12,7 +12,10 @@ About 4 hours 45 minutes
 - [UI/UX lesson](/ui-ux-design/ui-ux-design.md)
 
 ### Motivation
-Bootstrap is a popular and easy-to-use front-end framework first released by Twitter. Utilizing Bootstrap significantly reduces the amount of time spent styling your webpages, especially for mobile-development.
+Bootstrap is a popular and easy-to-use front-end framework first released by Twitter. Utilizing Bootstrap significantly reduces the amount of time spent styling your webpages, especially for mobile-development. 
+**Which companies use bootstrap?**
+- Intel
```

Comment:
provide links to examples. How do you know they use bootstrap? Where/how do they specifically use it?

Reply:
@alodahl I have searched for the examples but couldn't find anything specific. Just found the companies that use bootstrap and hence couldn't mention the description. Is it necessary to add the example?

Reply:
Yes.

Which web pages are they using bootstrap for?  If you cant figure it our, choose different companies.

Reply:
Okay.

Reply:
@alodahl can we add examples of websites made using bootstrap instead of companies?